composer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "davidandvinai/preferencesinfocommand",
```

Comment:
Should we add the `module-` prefix to the package name, that is `davidandvinai/module-preferencesinfocommand`?
That's the convention the core team uses, I've adopted it for my modules, too. What do you think?